scripts/pack_common.sh

```diff
@@ -20,7 +20,11 @@ set -e
 
 function get_stdcpp_lib()
 {
-    libname=`ldd ${BUILD_LATEST_DIR}/output/bin/pegasus_server/pegasus_server 2>/dev/null | grep libstdc++`
+    if [[ $2 == "false" ]]; then
+        libname=`ldd ${BUILD_LATEST_DIR}/output/bin/pegasus_server/pegasus_server 2>/dev/null | grep libstdc++`
+    else
+        libname=`ldd ${BUILD_LATEST_DIR}/output/bin/pegasus_meta_server/pegasus_meta_server 2>/dev/null | grep libstdc++`
```

Comment:
pegasus_meta_server and pegasus_replica_server depends on different libs, it's needed to find both of the servers' dependencies.

Reply:
Oh, I made a mistake, this is not a problem.